lib/env.mjs

```diff
@@ -65,13 +65,7 @@ export default createEnv({
 		GITHUB_CLIENT_ID: z.string().min(1),
 		GITHUB_CLIENT_SECRET: z.string().min(1),
 		NEXTAUTH_SECRET: z.string().min(1),
-		NEXTAUTH_URL: z.preprocess(
-			// This makes Vercel deployments not fail if you don't set NEXTAUTH_URL
-			// Since NextAuth.js automatically uses the VERCEL_URL if present.
-			str => process.env.VERCEL_URL ?? str,
-			// VERCEL_URL doesn't include `https` so it cant be validated as a URL
-			process.env.VERCEL ? z.string() : z.string().url()
-		),
+		NEXTAUTH_URL: z.string().url(),
```

Comment:
alternatively could do `str => 'https://${process.env.VERCEL_URL}' ?? str`

Reply:
thoughts?

Reply:
I don't understand but ITYJH

Reply:
we need an env var that represents the app url (so that we can show links to dashboard in the GitHub tracking comments)
Vercel URL is not always "maige.app" in prod. So this stops overwriting NextAuth URL